js/background.js

```diff
@@ -1340,6 +1340,18 @@
       }
     });
 
+    Whisper.events.on('devicePairingRequestReceivedNoListener', async () => {
+      window.pushToast({
```

Comment:
What would happen here if this event got triggered multiple times?

Reply:
Toasts with a distinct id will not appear more than once on the screen simultaneously

Reply:
Ohh 👍